mne/viz/evoked.py

```diff
@@ -479,19 +480,33 @@ def _plot_image(data, ax, this_type, picks, cmap, unit, units, scalings, times,
 
     # Set amplitude scaling
     data = this_scaling * data[picks, :]
-    im = ax.imshow(data, interpolation='nearest', origin='lower',
+    if ylim is None or this_type not in ylim:
```

Comment:
If you don't want to enforce symmetric limits you could reuse my `center_cmap` function in the ERDS example: http://www.martinos.org/mne/dev/auto_examples/time_frequency/plot_time_frequency_erds.html

Reply:
In principle that might be a cool thing, but we should probably do it more broadly than just in one function. I.e., not this PR.